control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -112,6 +112,16 @@ func (r Reconciler) ReconcileKind(ctx context.Context, cg *kafkainternals.Consum
 
 func (r Reconciler) FinalizeKind(ctx context.Context, cg *kafkainternals.ConsumerGroup) reconciler.Event {
 
+	cg.Spec.Replicas = pointer.Int32Ptr(0)
```

Comment:
could we have some unit test on the `TestFinalizeKind` for this file?

Reply:
@matzew https://github.com/knative-sandbox/eventing-kafka-broker/pull/2387/files#diff-ea2830fdd9c809584dd8d32aca8e1e011461c820dd262b02a59ce52c72f38fd1R1431 <- added some changes to it in this PR now